travel-planner/travel-planner-backend/models/maps.js

```diff
@@ -0,0 +1,92 @@
+const Parse = require("parse/node");
+const got = require("got");
+const { pipeline } = require("stream");
+
+class Maps {
+	static async createNewMap(title, center, sessionToken) {
+		const Maps = Parse.Object.extend("Maps");
+		const Timelines = Parse.Object.extend("Timelines");
+
+		const map = new Maps();
+
+		let query = new Parse.Query("_Session");
+		query.equalTo("sessionToken", sessionToken);
+		query.first().then(function (session) {
+			if (session) {
+				const user = session.get("user");
+				map.set("User", user);
+			} else {
+				throw "ERROR: Invalid session token!";
+			}
+		});
+
+		map.set("MapName", title);
+		map.set(
+			"Center",
+			new Parse.GeoPoint({ latitude: center.lat, longitude: center.lng })
+		);
+
+		map.set("Timeline", new Timelines());
+
+		map.save().then((newMap) => {
+			return newMap;
+		});
+	}
+
+	static async getUserMaps(sessionToken) {
+		let query = new Parse.Query("_Session");
+		query.equalTo("sessionToken", sessionToken);
+		query.first().then(function (session) {
+			if (session) {
+				const user = session.get("user");
+
+				query = new Parse.Query("Maps");
+				query.equalTo("User", user);
+				query
+					.find()
+					.then(function (maps) {
+						console.log(maps);
```

Comment:
Is this for testing? Do we want to remove it or keep it for debugging purpose when merging into master?

Reply:
This will be removed in the next pr. This was for debugging purposes and also to see what the maps object contains to use it in other functions.